EASY_INSTALL.m

```diff
@@ -1,12 +1,14 @@
 %% EASY INSTALL
 
-cd mid/mid/mxml-2.6/
+origin = pwd;
+mxml = [origin, '/mid/mid/mxml-2.6/'];
 
-system('make', '-echo');
-system('make install', '-echo');
-cd ../..
+system(['cd ', mxml, '; make'], '-echo');
+system(['cd ', mxml, '; make install'], '-echo');
 
-cd mid
+mid = [origin, '/mid/mid/'];
+
+cd mid/mid
```

Comment:
If `origin = fileparts(mfilename('fullpath'));`, this line would need to be changed to `cd(strcat(origin, '/mid/mid/'));`.